app/helpers/statistics_user_resources_helper.rb

```diff
@@ -0,0 +1,90 @@
+module StatisticsUserResourcesHelper
+  GROUP_LIST = Group.pluck :name, :id
```

Comment:
để biến @ nhé

Reply:
@groups ||= Group.pluck :name, :id

Reply:
tương tự. dùng ||= để tránh query nếu object đã tồn tại

Reply:
e thấy dùng biến @ xong nó toàn ra nil

Reply:
vs cả e tưởng biến của mình load lại mỗi khi vào trang này, để || như kia e nghĩ là nó load 1 lần sau mà mình update thì nó ko update lại @ngocvt-0484